src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -1525,5 +1546,26 @@ private static Common.InputAction MapGlfwInputAction(InputAction action)
                     throw new ArgumentOutOfRangeException(nameof(action), action, null);
             }
         }
+
+        private static CursorShape MapStandardCursorShape(MouseCursor.StandardShape shape)
+        {
+            switch (shape)
+            {
+                case MouseCursor.StandardShape.Arrow:
+                    return CursorShape.Arrow;
+                case MouseCursor.StandardShape.IBeam:
+                    return CursorShape.IBeam;
+                case MouseCursor.StandardShape.Crosshair:
+                    return CursorShape.Crosshair;
+                case MouseCursor.StandardShape.Hand:
+                    return CursorShape.Hand;
+                case MouseCursor.StandardShape.HResize:
+                    return CursorShape.HResize;
+                case MouseCursor.StandardShape.VResize:
+                    return CursorShape.VResize;
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(shape), shape, null);
+            }
+        }
```

Comment:
Do we need to do this mapping and/or have two separate types?

Reply:
The alternative would be a massive if else chain of `cursor == MouseCursor.IBeam`, so I'm not sure that's preferable.